### ISSUE TITLE
Seemingly random ignition failure - Not enough EC even when there is plenty

### ISSUE THREAD

**Issue by zorg2044** [linuxgurugamer/EngineIgnitor#12]
Hi @linuxgurugamer I've been getting some apparently random failures to ignite. The message is not enough EC even though there is plenty in reserve. The engines were "stable" each time I tried.

Tested with several different engines.

I have a log file here. Launch game, send to orbit via Alt-f12, a couple of succesful ignitions, followed by a failure with tons of spare EC

https://www.dropbox.com/s/pl6405rnla8rlah/output_log.txt?dl=0

Windows 10
KSP 1.7.3

![screenshot41](https://user-images.githubusercontent.com/39182212/66895564-931dfa80-f00c-11e9-959e-156d328c26e4.png)

One of the four engines failed here

I had a thought it might have been a clash with Dynamic Battery Storage (which is the only thing touching EC on this test install but this happened even with DBS uninstalled as with the log above).

Anyway hope you can get to the bottom of this! I have made a blanket patch for myself that removes the EC requirement for now but thats probably not an ideal solution in general.

EDIT: The catch all patch to remove EC requirements was included as an optional patch in a previous PR I did that was merged.

**Comment by linuxgurugamer** [comment]
is this still a problem?

**Comment by zorg2044** [comment]
I'm not sure tbh. in a previous PR I included an optional patch in a disabled file to remove EC requirements from all configs but since I did that I've mostly only been using our own parts in BDB with EI where we dont include EC requirements to begin with. 

It is worth considering removing EC requirement entirely. The ignition resources in EI are heavily abstracted so I dont necessarily see the value of having it.

Higher EC requirements for ignitions can add realism to a degree by representing spark ignitors and such but again as above doesnt seem to fit the abstracted philosophy of EI anyway. People who need that sort of thing have full fledged Real Fuels.

**Comment by kobe-maru** [comment]
I agree with @zorg2044 on the idea of removing EC requirement as a part of this entirely. We already removed the "hypergolic fluid" resource that was a redundant abstraction of ignitors anyway. 

The only real benefit I see from having an EC check is to make sure the vessel is controllable before engine start up, but to some degree that is also covered by other mods and now stock as well (rt and commnet).
